e2e/tools/validator/metric_validations.yaml

```diff
@@ -0,0 +1,319 @@
+config:
+  mapping:
+    actual: latest
+    predicted: dev
+
+validations:
+  # absolute power comparison
+  - name: Total - absolute
```

Comment:
Can we also validate these invariants in the same version of `dev` 

* kepler_node_<pkg|core|uncore|dram|other>{dev} = sum of ( process_<pkg|core|uncore|dram|other>{dev} ) 
* kepler_node_<pkg|core|..> = node_exporter_rapl_<pkg|core...>
*sum( kepler_process_bpf_cpu ) = node_exporter_cpu_time

Reply:
for `kepler_node_<pkg|core|dram...>{dev} = sum of (process_<pkg|core|dram....>){dev}` do you mean 
MAE of `sum(rate(kepler_node<pkg|core|dram>){dev}[20s]) and sum(rate(process_<pkg|core|dram>{dev}[20s]))`?